src/saturn_engine/utils/tester/json_utils.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+from typing import Tuple
+
+
+def find_node(obj: dict, val: str, path: list[str]) -> Optional[list[str]]:
+    if isinstance(obj, dict):
+        for k, v in obj.items():
+            if isinstance(v, str) and v == val:
```

Comment:
We could just type `val` to `object` and do `if v == val`, no need to limit ourself to `str`.